src/react-components/settings-menu.js

```diff
@@ -89,14 +97,21 @@ export default class SettingsMenu extends Component {
                   </i>
                 </div>
                 <div className={styles.listItem}>
-                  <StateLink
-                    stateKey="modal"
-                    stateValue="rename_room"
-                    history={this.props.history}
-                    onClick={this.props.hideSettings}
+                  <a
+                    href="#"
+                    onClick={() =>
```

Comment:
Went on a pedantry google journey when I saw this. We use this `<a href="#" onClick=...` because we want buttons that look like links but don't actually link to anything. In fact the "Choose a Scene" link/button just above this is just a `<div>`.

I only care in this case because you forgot to `preventDefault` in the link variant here.

Ideally, we should be using `<button>`s here, that are styled like links. But I would be okay with using a `<span>` instead, so that we don't have to worry about `href` and `preventDefault`, and it's annoying to have to reset browser button styles just to make them look like links.

Reply:
I'll list some benefits of semantic HTML while I'm in rumination mode:
- Consistency and predictability within our own code base. Especially useful when styling elements to be able to apply styles consistently on particular element types.
- Accessibility, so that screen readers know how to read our app. (Arguably we could just put `role`/ARIA attributes on everything instead, but why do that when we get it for free if we are semantic from the start)
- Generally being nice to machines that might want to understand our app for whatever reason.
- Predictability for people that might attempt to write user scripts against our UI.

Reply:
Yup, I actually had already replaced a few of these with buttons on my next branch :)